gradle.properties

```diff
@@ -4,12 +4,6 @@ org.gradle.parallel=true
 org.gradle.caching=true
 # mandatory from agp 3.6
 android.useAndroidX=true
-# we use testFixtures, and it has problems when project includes android modules
-# temporary set this to false in `~/.gradle/gradle.properties` to work with modules that uses testFixtures as a workaround
-# https://issuetracker.google.com/issues/139438142
```

Comment:
Should we keep knowledge about this decision somewhere?

Reply:
I see, https://github.com/avito-tech/avito-android/pull/298#issuecomment-609077272
Tech radar is good for me.
We can write it later in case of new questions.